### DISCUSSION TITLE
NOT working on Kubuntu 22.04

### DISCUSSION THREAD

**Discussion by pigeonskiller** [AppOutlet/AppOutlet#264] [Q&A] [answered]
Hi, sorry to inform you it does NOT work Kubuntu 22.04.

Same behavior both with double click and from terminal.

Here the log from terminal:

./App.Outlet-2.1.0.AppImage
(node:9755) electron: The default of contextIsolation is deprecated and will be changing from false to true in a future release of Electron.  See https://github.com/electron/electron/issues/23506 for more information
query: PRAGMA foreign_keys = OFF
query: BEGIN TRANSACTION
query: SELECT null as database, null as schema, * FROM "sqlite_master" WHERE "type" = 'table' AND "name" IN ('application')
query: SELECT null as database, null as schema, * FROM "sqlite_master" WHERE "type" = 'table' AND "name" IN ('settings')
query: SELECT null as database, null as schema, * FROM "sqlite_master" WHERE "type" = 'index' AND "tbl_name" IN ('application')
query: SELECT null as database, null as schema, * FROM "sqlite_master" WHERE "type" = 'index' AND "tbl_name" IN ('settings')
query: PRAGMA table_info("application")
query: PRAGMA index_list("application")
query: PRAGMA foreign_key_list("application")
query: PRAGMA table_info("settings")
query: PRAGMA index_list("settings")
query: PRAGMA foreign_key_list("settings")
query: PRAGMA index_info("sqlite_autoindex_settings_1")
query: SELECT * FROM "sqlite_master" WHERE "type" = 'table' AND "name" = 'typeorm_metadata'
query: COMMIT
query: PRAGMA foreign_keys = ON
query: SELECT "Settings"."id" AS "Settings_id", "Settings"."key" AS "Settings_key", "Settings"."value" AS "Settings_value" FROM "settings" "Settings" WHERE "Settings"."key" = ? LIMIT 1 -- PARAMETERS: ["LAST_SYNC"]
No need to synchronize
[9755:0523/155641.928359:FATAL:gpu_data_manager_impl_private.cc(445)] GPU process isn't usable. Goodbye.
Trace/breakpoint trap (core dumped)

Hope you solve the bug soon. Thanks for the app.

**Comment by BenHerbst** [comment]
Same issue on Pop!_OS 22.04. Using NVIDIA propriety drivers. Using Wayland

**Comment by BenHerbst** [comment, marked as answer]
You can launch via app-outlet --disable-gpu-sandbox on Nvidia.
Its a bug with electron framework and Nvidia on Linux

**Reply by MessiasLima** [reply]
@pigeonskiller could you please test it and mark as resolved if so?

**Reply by pigeonskiller** [reply]
Yes, thanks. Now working using '--disable-gpu-sandbox' option, but not all icons are loaded.

**Reply by MessiasLima** [reply]
About the GPU bug: It will probably be solved in the next release because I'm going to update some dependencies.
About the Icons: It is not related to the App Outlet directly. You are talking about AppImage icons, aren't you? https://github.com/AppImage/appimage.github.io/issues/2800

**Reply by pigeonskiller** [reply]
Checked. AppImage icons are very slow to show or not loaded. Thanks.
